src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CommonFunction.java

```diff
@@ -136,4 +136,8 @@ private static boolean isConsentedApprovedOrderCollectionPresent(Map<String, Obj
     private static boolean isContestedApprovedOrderCollectionPresent(Map<String, Object> caseData) {
         return caseData.get(CONTESTED_CONSENT_ORDER_COLLECTION) != null && !((List<Map>) caseData.get(CONTESTED_CONSENT_ORDER_COLLECTION)).isEmpty();
     }
+
+    public static boolean isDocumentPresentInCaseData(String documentName, CaseDetails caseDetails) {
```

Comment:
not sure this method really offers us anything. it shouldn't be specific to documents as the logic would work for any case data field. I think just call containsKey method directly

Reply:
still don't think this offers us anything.